src/code_8057824_1.c

```diff
@@ -3557,3 +3699,1445 @@ void sub_808B50C(void)
   sub_803E708(10,70);
   gDungeonGlobalData->unk2 = 1;
 }
+
+void JirachiWish(void)
+{
+  u8 friendArea;
+  struct DungeonEntity *JirachiEntity;
+  struct Position *LeaderPos;
+  struct DungeonEntity *LeaderEntity;
+  s32 wishChoice;
+  s32 iVar4;
+  u32 uVar6;
+  s32 iVar9;
+  struct ItemSlot auStack152 [9];
+  struct ItemSlot auStack116 [9];
+  struct ItemSlot auStack80 [9];
+  s32 local_2c;
+  s32 local_28;
+  s32 local_24;
+  s32 temp;
+  s32 temp2;
+  s32 temp3;
+
+  JirachiEntity = GetEntityFromClientType(0x1a);
+  CopySpeciesNametoBuffer(gUnknown_202E038,0x19d); // SPECIES_JIRACHI
```

Comment:
This can use the `SPECIES_JIRACHI` macro instead of just a comment.

Reply:
Done